tuna/export_db.py

```diff
@@ -250,20 +251,21 @@ def write_fdb(arch, num_cu, ocl, find_db, filename=None):
   FDBRecord = namedtuple('FDBRecord',
                          'alg_lib solver_id kernel_time workspace_sz')
 
-  with open(file_name, 'w') as out:
+  with open(file_name, 'w') as out:  # pylint: disable=unspecified-encoding
     for key, solvers in sorted(find_db.items(), key=lambda kv: kv[0]):
       solvers.sort(key=lambda x: float(x[2]))
       lst = []
       # for alg_lib, solver_id, kernel_time, workspace_sz in solvers:
       for rec in solvers:
         rec = FDBRecord(*rec)
+        # pylint: disable-next=consider-using-f-string ; more reable
```

Comment:
looks like u addressed this, is this still necessary?

Reply:
it's not addressed -- :262 doesn't use a f-string